packages/library-base/src/commonMain/kotlin/io/realm/notifications/ListChangeImpl.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.notifications
```

Comment:
I think all of these `*Impl` classes should be in `io.realm.internal` since they otherwise creep into the public API on Java.

Reply:
They cannot be in a different package than the sealed interfaces.